spec/unit/inspect_task_spec.rb

```diff
@@ -40,6 +54,9 @@ def inspect(_system, description, _filter, _options = nil)
       result = SimpleInspectTaskScope.new("bar" => "baz")
 
       description.bar = result
+    end
+
+    def summary(description)
```

Comment:
Unused method argument - `description`. If it's necessary, use `_` or `_description` as an argument name to indicate that it won't be used. You can also write as `summary(*)` if you want the method to accept any arguments but don't care about them.